tests/integration/test_integration.py

```diff
@@ -47,10 +48,12 @@ async def test_given_charm_is_built_when_deployed_then_status_is_active(
     )
 
 
-async def test_given_tls_requirer_is_deployed_and_related_then_certificate_is_created_and_passed_correctly(  # noqa: E501
-    ops_test,
+async def test_given_charm_is_scaled_when_tls_requirer_is_related_then_certificate_is_created_and_passed_correctly(  # noqa: E501
+    ops_test: OpsTest,
```

Comment:
Would it make sense to split these 2 cases into separate tests?

Reply:
Done